src/app.js

```diff
@@ -29,8 +41,8 @@ app.get("/notes", async (req, res) => {
     try {
         let notes;
 
-        if (req.query.userid) {
-            notes = await getAllUserNotes(req.query.userid);
+        if(req.query.userid && req.query.is_deleted){
+            notes = await getAllUserNotes(req.query.userid, req.query.is_deleted);
```

Comment:
I suggest adding another if-else that can take JUST userid and not have is_deleted. Also recommend changing the query parameter name to be just one word, like "deleted"